docs/source/connecting_datasource.rst

```diff
@@ -15,37 +15,24 @@ To register your data source to MLOps, you must have the provider credentials ac
 
```

Comment:
You need to reference this file in `docs/source/starters.rst` for it to show

Reply:
https://github.com/datarisk-io/mlops_codex/pull/68/commits/143dbfaccb9bd83c20f413971eefb6ad122f3163